Gir_Gio.toml

```diff
@@ -0,0 +1,30 @@
+[options]
+girs_dir = "gir-files"
+library = "Gio"
+version = "2.0"
+min_cfg_version = "2.32"
+target_path = "."
+work_mode = "normal"
+
+generate = [
+    "Gio.Action",
+    "Gio.ActionGroup",
+    "Gio.ActionMap",
+    "Gio.AppInfo",
+    "Gio.AppLaunchContext",
+    "Gio.Application",
+    "Gio.Menu",
+    "Gio.MenuAttributeIter",
+    "Gio.MenuItem",
+    "Gio.MenuLinkIter",
+    "Gio.MenuModel",
+    "Gio.Permission",
+    "Gio.Settings",
+    "Gio.SimpleAction",
+    "Gio.SimpleActionGroup",
+    "Gio.SimplePermission",
+]
```

Comment:
Not sure if it works on Windows but
    "Gio.Drive",
    "Gio.Mount",
    "Gio.Volume",
    "Gio.VolumeMonitor",